quaistats/quaistats.go

```diff
@@ -524,10 +524,13 @@ func (s *Service) handleBlock(block *types.WorkObject) {
 			s.transactionStatsQueue.Enqueue(txStats)
 		}
 	}
-
 	// After handling a block and potentially adding to the queues, notify the sendStats goroutine
 	// that stats are ready to be sent
-	s.statsReadyCh <- struct{}{}
```

Comment:
I thought this block was intentional, so you don't loop back around and start modifying stats data while its being shipped out or something like that. Are you sure this is ok?

Reply:
I did not remove this block. I simply changed it to a non-blocking send. If the channel consumer is ready to receive something, it is received, but if the channel consumer is ignoring us or is not ready for whatever reason, we don't need to block stats to wait for it.

Reply:
yeah, it was a block, not its non-blocking. I believe it _should_ be blocking